onnx_tf/handlers/backend/bernoulli.py

```diff
@@ -0,0 +1,31 @@
+import tensorflow as tf
+from tensorflow_probability import distributions as tfd
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import partial_support
+from onnx_tf.handlers.handler import ps_description
+
+
+@onnx_op("Bernoulli")
+@partial_support(True)
+@ps_description(
+    "Bernoulli with float type seed will be converted to int type seed")
+class Bernoulli(BackendHandler):
+
+  @classmethod
+  def _common(cls, node, **kwargs):
+    x = kwargs["tensor_dict"][node.inputs[0]]
+    dtype = x.dtype
+    if 'dtype' in node.attrs:
+      dtype = node.attrs.get('dtype')
```

Comment:
Could be combined to one line, dtype = node.attrs.get("dtype", x.dtype)

Reply:
Thanks, updated.